src/browser/api/window.js

```diff
@@ -178,6 +178,9 @@ let optionSetters = {
                 });
                 browserWin.setResizeRegion(resizeRegion.size);
                 browserWin.setResizeRegionBottomRight(resizeRegion.bottomRightCorner);
+                if (resizeRegion.sides) {
```

Comment:
Don't think there's a need to check if `resizeRegion.sides` is defined. If it is `''`, then it'll do nothing, correct?

Reply:
Add the default value for `sides` here: https://github.com/HadoukenIO/core/pull/267/files#diff-d5752a5b2c3b7ce27c77d78f1b9278a6R175